unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/EntityList/BuildModeEntityListController.cs

```diff
@@ -0,0 +1,78 @@
+using DCL.Controllers;
+using DCL.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class BuildModeEntityListController : MonoBehaviour
+{
+    public enum EntityAction
+    {
+        SELECT = 0,
+        LOCK = 1,
+        DELETE = 2,
+        SHOW = 3
+    }
+
+    public System.Action<DecentralandEntityToEdit> OnEntityClick,OnEntityDelete,OnEntityLock,OnEntityChangeVisibility;
```

Comment:
```suggestion
    public System.Action<DecentralandEntityToEdit> OnEntityClick;
    public System.Action<DecentralandEntityToEdit> OnEntityDelete;
    public System.Action<DecentralandEntityToEdit> OnEntityLock;
    public System.Action<DecentralandEntityToEdit> OnEntityChangeVisibility;
```
We dont have a policy against single line declarations, but I think this is more readable.